src/plugins/auth/ldap.js

```diff
@@ -114,7 +114,7 @@ function ldapAuth(manager, client, user, password, callback) {
 	// auth plugin API
 	function callbackWrapper(valid) {
 		if (valid && !client) {
-			manager.addUser(user, null);
+			manager.addUser(user, null, true);
```

Comment:
Could you remove `|| false` [here](https://github.com/thelounge/thelounge/blob/a24ab74c4cca262ba9dc516e1acf9493957e033a/src/clientManager.js#L115) please?
The only place that was not sending it as an argument was this one, so we can have no fallback now.

Reply:
I have now added this